File: cpp/38a_1873B_GoodKid.cpp
#include<bits/stdc++.h>

int main(){
    int t, n, temp, cnt;
    std::vector<int> ans;
    std::cin>>t;
    for (int i = 0; i < t; i++){
        std::cin>>n;
        std::vector<int> arr;
        cnt = 1;
        for(int j = 0; j < n; j++){
            std::cin>>temp;
            arr.push_back(temp);
        }
        sort(arr.begin(), arr.end());
        arr[0] = arr[0] + 1;
        for (auto j : arr){
            cnt *= j;
        }
        ans.push_back(cnt);
    }
    for (auto i : ans){
        std::cout<<i<<"\n";
    }
    return 0;
}

File: python/38a_1873B_GoodKid.py
def main():
    ans = []
    t = int(input())
    for _ in range(t):
        n = int(input())
        arr = input().split()
        for i in range(n):
            arr[i] = int(arr[i])
        arr.sort()
        arr[0] = arr[0] + 1
        cnt = 1
        for i in arr:
            cnt *= i
        ans.append(cnt)
    for i in ans:
        print(i)

if __name__ == "__main__":
    main()